kong/db/schema/init.lua

```diff
@@ -1959,7 +1963,7 @@ function Schema:process_auto_fields(data, context, nulls, opts)
 
       if self.shorthand_fields then
         for _, shorthand_field in ipairs(self.shorthand_fields) do
-          if shorthand_field[key] and shorthand_field[key].translate_backwards then
+          if shorthand_field[key] and shorthand_field[key].deprecation and shorthand_field[key].deprecation.replaced_with and #shorthand_field[key].deprecation.replaced_with > 0 then
```

Comment:
See above comment, perhaps this could then be changed to:

```suggestion
          if shorthand_field[key] and replaced_with then
```

Reply:
Done ✅